app/views/SupportForCoronavirusView.scala.html

```diff
@@ -0,0 +1,44 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+
+@this(
+    main_template: MainTemplate,
+    appConfig: AppConfig)
+
+@()(implicit request: Request[_], messages: play.api.i18n.Messages)
+
+@main_template(title = messages("support.for.coronavirus.title")) {
+
+    @components.back_link()
+
+    <h1 class="header-margin-top">@messages("support.for.coronavirus.title")</h1>
+
+    <p>@messages("support.for.coronavirus.p1")</p>
+
+    <li>
```

Comment:
<li> elements should be wrapped within <ul class="list list-bullet"> 
This will also resolve the non-spacing issue between the last bullet point and the dynamically generated Nuance link